codecov.yml

```diff
@@ -1,6 +1,7 @@
 codecov:
   branch: master
-
+  notify:
+    after_n_builds: 10
```

Comment:
this is so that codecov doesn't report on insufficient coverage prematurely

Reply:
is this related to why codecov often gives an Access Denied?

Reply:
Without this, then Codecov will report coverage as soon as one job finishes. But if that job isn't comprehensive, then it'll show e.g. "coverage 30% less than target of 80%, failed". Like this, it'll wait for multiple jobs to be done and will merge their reports